packages/core/middleware/middleware.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
```

Comment:
shouldn't whole `middleware` dir be in `nuxt` dir?

Reply:
added module to `/nuxt` directory and middleware will stay in the root - that's for the future if we want to use middleware as separated app